app/src/main/java/com/example/voyagerx/ui/fragments/landing/list/LaunchOverviewViewHolder.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.voyagerx.ui.fragments.landing.list
+
+import androidx.recyclerview.widget.RecyclerView
+import com.example.voyagerx.databinding.LandingPageOverviewCardBinding
+import com.example.voyagerx.ui.fragments.landing.data.LaunchOverviewData
+
+class LaunchOverviewViewHolder(private val binding: LandingPageOverviewCardBinding) :
+    RecyclerView.ViewHolder(binding.root) {
+
+    fun bind(item: LaunchOverviewData) {
```

Comment:
Would this logic be different if we used Launch instead of LaunchOverviewData?

Reply:
I don't think it would be different, after talking to Celina I'll refactor this to use Launch